tests/model_registry/utils.py

```diff
@@ -156,3 +161,71 @@ def get_model_registry_db_label_dict(db_resource_name: str) -> dict[str, str]:
         Annotations.KubernetesIo.INSTANCE: db_resource_name,
         Annotations.KubernetesIo.PART_OF: db_resource_name,
     }
+
+
+def get_pod_container_error_status(pod: Pod) -> str:
+    pod_instance_status = pod.instance.status
+    # Check the containerStatuses and if any containers is in waiting state, return that information:
```

Comment:
please add docsting and remove the comment.
why is this function needed? you can directly use `container_status.get("state", {}).get("waiting")`

Reply:
I would be keeping the function. It is cleaner code this way. Added the comments for code maintainability.